bitnami/rabbitmq-cluster-operator/templates/messaging-topology-operator/metrics-service.yaml

```diff
@@ -49,5 +49,4 @@ spec:
     {{- end }}
   selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
     app.kubernetes.io/component: messaging-topology-operator
-    app.kubernetes.io/part-of: rabbitmq
```

Comment:
Why didd you remove this label?

Reply:
I deleted that selector label, because in the deployment, the spec.template.metadata.label does not contain this so as a result no pods are found and a service is created with no pods assigned.